__tests__/app.articles.test.js

```diff
@@ -0,0 +1,57 @@
+const request = require("supertest");
+const app = require("../app");
+
+const testData = require("../db/data/test-data/index");
+const seed = require("../db/seeds/seed");
+const db = require("../db/connection");
+
+beforeEach(() => {
+  return seed(testData);
+});
+afterAll(() => {
+  return db.end();
+});
+
+describe("GET", () => {
+  test(`200: /api/articles/:article_id - 
+    Responds with an article object, which has properties:
+        "author" which is the "username" from the users table
+        "title"
+        "article_id"
+        "body"
+        "topic"
+        "created_at"
+        "votes"`, () => {
+    return request(app)
+      .get("/api/articles/4")
+      .expect(200)
+      .then(({body}) => {
+        expect(body).toEqual(
+          expect.objectContaining({
+            title: expect.any(String),
+            article_id: expect.any(Number),
```

Comment:
You need to check you've got the article with the id that you requested. At the moment, this would pass for any article you get back.